deploy/olm-catalog/kogito-operator/0.13.0/kogito-operator.v0.13.0.clusterserviceversion.yaml

```diff
@@ -762,17 +762,14 @@ spec:
       name: keycloaks.keycloak.org
       version: v1alpha1
   description: |-
-    Kogito Operator is designed to deploy Kogito services from source (only on OpenShift) and every piece of infrastructure that the services might need:
+    Kogito Operator is designed to deploy Kogito Runtimes Services from source and every piece of infrastructure that the services might need:
 
-    * Creates two kinds of build configuration: Native or JVM. For more information refer to [Native build and deployment](https://docs.jboss.org/kogito/release/latest/html_single/#_native_build_and_deployment) section on our docs
-    * Builds a Kogito service from a remote git repository and creates a custom image for the given runtime: Quarkus or Springboot. [See more](https://docs.jboss.org/kogito/release/latest/html_single/#con-kogito-operator-deployment-options_kogito-deploying-on-openshift).
-    * Installs the [Kogito Data Index Service](https://docs.jboss.org/kogito/release/latest/html_single/#con-kogito-operator-with-data-index-service_kogito-deploying-on-openshift) to enable data index feature for Kogito services.
-    * Installs the [Kogito Jobs Service](https://docs.jboss.org/kogito/release/latest/html_single/#con-kogito-operator-with-jobs-service_kogito-deploying-on-openshift) to enable jobs scheduling.
+    * Creates two kinds of build configuration: Native or JVM. For more information refer to [Native X JVM Builds](https://github.com/kiegroup/kogito-cloud-operator#native-x-jvm-builds) on our docs
+    * Builds a Kogito Runtime Service from a remote git repository and creates a custom image for the given runtime: Quarkus or Springboot. [See more](https://github.com/kiegroup/kogito-cloud-operator#kogito-runtimes-service-deployment).
+    * Installs the [Kogito Data Index Service](https://github.com/kiegroup/kogito-runtimes/wiki/Data-Index-Service) to enable data index feature for Kogito Runtimes. [See more](https://github.com/kiegroup/kogito-cloud-operator#kogito-data-index-service-installation).
+    * Installs the [Kogito Jobs Service](https://github.com/kiegroup/kogito-runtimes/wiki/Jobs-Service) to enable jobs scheduling.
```

Comment:
This is old, from where you took this CSV?

Reply:
this was generated into the manifests folder by the `make vet` ... gonna check ...

Reply:
https://github.com/kiegroup/kogito-cloud-operator/blob/f4c3e4f1b7c58df829b2ebee9b99aba911bfa02e/deploy/olm-catalog/kogito-operator/manifests/kogito-operator.clusterserviceversion.yaml#L767

Reply:
I think the manifests csv file was never updated by hand ... only the versioned file.
We should enforce the manifests folder to be changed and then use `make vet`
Copying these 0.12 changes to manifests folder